### PR TITLE
feature: roadwork detail page

### PR DESCRIPTION
**Pull request by DeannaSharma**

### CODE REVIEW

File: site/utils/index.ts [hunk 1]
```diff
@@ -214,3 +214,9 @@ export const getStopsForRoutes = async (
         )
     ).flat();
 };
+
+export const toTitleCase = (text: string) => {
+    return text.replace(/([^\W_]+[^\s-]*) */g, function (txt) {
```
Comment (alainajaffer) [inline site/utils/index.ts:219]: could we use an arrow function here?

File: site/utils/index.ts [hunk 2]
```diff
@@ -214,3 +214,9 @@ export const getStopsForRoutes = async (
...
+        return txt.charAt(0).toUpperCase() + txt.substring(1).toLowerCase();
```
Comment (alainajaffer) [inline site/utils/index.ts:220]: could this be defensive what if txt is empty?

File: site/utils/index.test.ts [hunk 3]
```diff
@@ -133,3 +133,15 @@ describe("filterDisruptionsForOperatorUser", () => {
...
     });
 });
+
+describe("toTitleCase", () => {
+    it.each([
+        ["KING STREET", "King Street"],
+        ["king street", "King Street"],
+        ["HYPHENATED-STREET", "Hyphenated-Street"],
+        ["hyphenated-street", "Hyphenated-Street"],
+        ["RanDoM cAsE", "Random Case"],
```
Comment (alainajaffer) [inline site/utils/index.test.ts:143]: A case for empty string

Comment (DeannaSharma) [reply site/utils/index.test.ts:143]: added

File: site/utils/index.ts (hunk 2 above)
Comment (DeannaSharma) [reply site/utils/index.ts:220]: if empty string is passed an empty string is returned

**Review by alainajaffer — APPROVED** [review]

**Pull request merged**